src/main/scala/nl.knaw.dans.easy.bag2deposit/ddm/DdmTransformer.scala

```diff
@@ -17,16 +17,19 @@ package nl.knaw.dans.easy.bag2deposit.ddm
 
 import better.files.File
 import nl.knaw.dans.easy.bag2deposit.InvalidBagException
+import nl.knaw.dans.easy.bag2deposit.collections.Collections.getCollectionsMap
 import nl.knaw.dans.easy.bag2deposit.ddm.LanguageRewriteRule.logNotMappedLanguages
 import nl.knaw.dans.easy.bag2deposit.ddm.ReportRewriteRule.logBriefRapportTitles
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import org.apache.commons.configuration.PropertiesConfiguration
 
 import scala.util.{ Failure, Success, Try }
 import scala.xml.transform.{ RewriteRule, RuleTransformer }
 import scala.xml.{ Elem, Node, NodeSeq }
 
-case class DdmTransformer(cfgDir: File, collectionsMap: Map[String, Elem] = Map.empty) extends DebugEnhancedLogging {
+case class DdmTransformer(cfgDir: File, properties: PropertiesConfiguration = null) extends DebugEnhancedLogging {
```

Comment:
I did a quick try with a lazy argument. That doesn't mary with a case class required by the current unit tests
```suggestion
class DdmTransformer(cfgDir: File, properties: => PropertiesConfiguration = null) extends DebugEnhancedLogging {
```